modules/installation-supported-aws-machine-types.adoc

```diff
@@ -361,3 +361,87 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 
 |===
 ====
+
+.Instance types for arm64 machines
+[%collapsible]
+====
+[cols="2a,2a,2a,2a",options="header"]
+|===
+
+|Instance type
+|Bootstrap
+|Control plane
+|Compute
+
+|`m6g.large`
+|x
+|
+|x
+
+|`m6g.xlarge`
+|
+|x
+|x
+
+|`m6g.2xlarge`
+|
+|x
+|x
+
+|`m6g.4xlarge`
+|
+|x
+|x
+
+|`m6g.8xlarge`
+|
+|x
+|x
+
+|`m6g.12xlarge`
+|
+|x
+|x
+
+|`m6g.16xlarge`
+|
+|x
+|x
+
+|`c6g.large`
```

Comment:
AFAIK, `m6g.*` machine types had been tested by QE, not sure if `c6g.*` had been tested, please @shivanthi-amara help to confirm, thanks.

Reply:
@yunjiang29 - Yes, I tested with a  c6g.2xlarge cluster and also ran an E2E test. Here's the Jenkins job - https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/parallel-runner/7680/parameters/

The pass percentage is 91 %

Reply:
@shivanthi-amara thanks for confirming. 
Are there any machine-type-related bugs? If not, I think we can add `c6g.*` to the list.

Reply:
@kelbrown20 - I confirmed with Yunfei and we are good to merge this PR